units/check_blocked_tasks.go

```diff
@@ -54,6 +53,19 @@ func NewCheckBlockedTasksJob(distroId string, ts time.Time) amboy.Job {
 }
 
 func (j *checkBlockedTasksJob) Run(ctx context.Context) {
+	var tasksToCheck []task.Task
+	if j.DistroId != "" {
+		tasksToCheck = j.getTasksToCheckDistro()
+	} else {
+		tasksToCheck = j.getTasksToCheckContainer()
+	}
+	dependencyCache := map[string]task.Task{}
+	for _, t := range tasksToCheck {
+		j.AddError(checkUnmarkedBlockingTasks(&t, dependencyCache))
+	}
+}
+
+func (j *checkBlockedTasksJob) getTasksToCheckDistro() []task.Task {
```

Comment:
nit: getDistroTasksToCheck is more of a function name (and getContainerTasksToCheck)